### PR TITLE
Add license

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: agpl-3.0.pt-br.html [hunk 1]
```diff
@@ -0,0 +1,179 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
```
Comment (vitormattos) [inline agpl-3.0.pt-br.html]: I think that is only necessary to have the COPYING file instead of a html file.

Comment (ghost) [reply agpl-3.0.pt-br.html]: Hello Vitor! You are correct. The HTML file is an unofficial translation of AGPL license which I included here in the hope that this translation will help Brazilian Portuguese speakers understand the GNU AGPL better.

For reference, I am linking the Free Software Foundation opinion regarding translations of their licenses: https://www.gnu.org/licenses/translations.en.html

If the HTML format is a blocker, the file can be substituted by any of the following formats: Markdown, PDF, ODT
If including an unofficial translation is a blocker, the file can be removed with no harm.

I am eager to hear your opinion about that!

Comment (vitormattos) [reply agpl-3.0.pt-br.html]: My point is if is necessary to have two license files.

I like to good estimate the national language but, this is a too much internal and specific file. In this case, I think that maintain the industry pattern using English in all code is a good to follow. I believe that the unique necessary pattern is to identify the type of license, in this case AGPL, after this, speakers of each language can search by translations if want to read in your own language.

Comment (ghost) [reply agpl-3.0.pt-br.html]: Hi Vitor! I get your point. I will remove this extra file :)

**Review by vitormattos — APPROVED** [review]

**Pull request merged**